drivers/serial/uart_ns16550.c

```diff
@@ -464,6 +466,15 @@ static int uart_ns16550_configure(const struct device *dev,
 #endif
 		);
 
+	if ((INBYTE(IIR(dev)) & IIR_FE) == IIR_FE)
+#ifdef CONFIG_UART_NS16750
+		dev_data->fifo_size = 64;
+#else
+		dev_data->fifo_size = 16;
+#endif
+	else
+		dev_data->fifo_size = 1;
+
```

Comment:
Brackets are needed even for a one line statement.

Reply:
Done.